ci/cudf_pandas_scripts/run_tests.sh

```diff
@@ -67,15 +67,24 @@ fi
 python -m pip install ipykernel
 python -m ipykernel install --user --name python3
 
-# The third-party integration tests are ignored because they are run nightly in seperate CI job
+# The third-party integration tests are ignored because they are run in a separate nightly CI job
 python -m pytest -p cudf.pandas \
     --ignore=./python/cudf/cudf_pandas_tests/third_party_integration_tests/ \
+    --numprocesses=8 \
+    --dist=worksteal \
+    -k "not test_cudf_pandas_profiler" \
     --cov-config=./python/cudf/.coveragerc \
     --cov=cudf \
     --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cudf-pandas-coverage.xml" \
     --cov-report=term \
     ./python/cudf/cudf_pandas_tests/
 
+python -m pytest -p cudf.pandas \
```

Comment:
Can we please add a comment above this referencing Vyas's comment in this PR that running with 1 worker is necessary for profiler tests?

Reply:
Good call.